src/components/DirbTool/DirbTool.tsx

```diff
@@ -18,7 +18,7 @@ const description_userguide =
 
 interface FormValues {
```

Comment:
Should be FormValuesType. 
```suggestion
interface FormValuesType {
```